pkg/api/v1/types.go

```diff
@@ -2196,6 +2208,9 @@ type PodStatus struct {
 	// More info: http://kubernetes.io/docs/user-guide/pod-states#container-statuses
 	// +optional
 	ContainerStatuses []ContainerStatus `json:"containerStatuses,omitempty" protobuf:"bytes,8,rep,name=containerStatuses"`
+	// QOS class of the Pod
```

Comment:
How is this?

```
// The Quality of Service (QOS) classification assigned to the pod based on the  resource requests and limits
// See PodQOSClass type for available QOS classes
// https://github.com/kubernetes/kubernetes/blob/master/docs/design/resource-qos.md
```

Listing out the classification logic here would be a little too verbose I think.  Plus if the logic changes, this comment block is likely to be overlooked in an update.